### PR TITLE
Pilfer HH task

### PR DESCRIPTION
**Pull request by sennett**
Allows to transfer funds at a specific ERC-20 token address from theRich to thePoor.  All the funds are moved:

```
sennett@MBP multiply-proxy-actions % npx hardhat transferErc20 --network local --from 0x53c286e0abe87c9e6d4d95ebe62ceafa4afce849 --to 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --token 0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e
Forking from block number: 14473828
const fromSigner = await hre.ethers.getSigner(fromAddress)
const toSigner = await hre.ethers.getSigner(toAddress)

    from  = 0x53c286e0abe87c9e6d4d95ebe62ceafa4afce849
    to    = 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
    token = 0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e
    
 ----------------------------- 
| starting balances for token |
 ----------------------------- 
from: 1547379195672332123477
to  : 0
 --------------------------- 
| ending balances for token |
 --------------------------- 
from: 0
to  : 1547379195672332123477
 --------------------------- 
done
sennett@MBP multiply-proxy-actions %
```

### CODE REVIEW

**Comment by paszkowskiDamian** [comment]
Did you try to run it? Apparently params cannot contain uppercase letters

File: tasks/pilfer.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    const tokenAddress = await taskArgs.tokenAddress
+
+    // send the rich some eth so that she can cover the gas costs of the transaction
+    await thePoorSigner.sendTransaction({
+      from: thePoorAddress,
+      to: theRichAddress,
+      value: hre.ethers.utils.parseEther('1'),
+      gasLimit: hre.ethers.utils.hexlify(1000000),
+    })
```
Comment (paszkowskiDamian) [inline tasks/pilfer.ts]: I would send ether to theRich from one of the deafult test accounts, only those guarantee that have some eth to send.

Comment (peculiarity) [reply tasks/pilfer.ts]: In this implementation what guarantees that the poor also has balance?

**Review by peculiarity — APPROVED** [review]

File: tasks/transferErc20.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { task } from 'hardhat/config'
+import erc20abi from '../abi/IERC20.json'
+
+task('transferErc20', 'Transfers erc20 funds between two addresses.  Mainnet not supported :(')
+  .addParam('from', 'Address of the wallet that we are transferring tokens from (robbing the rich)')
+  .addParam('to', 'Address of the wallet that we are transferring tokens to (to feed the poor)')
+  .addParam(
+    'token',
+    'The address of the token that we want to transfer.  The entire balance at theRich address will be transferred to thePoor.',
```
Comment (paszkowskiDamian) [inline tasks/transferErc20.ts:9]: you have left `thePoor` and `theRich` in param description.

Comment (sennett) [reply tasks/transferErc20.ts:9]: @paszkowskiDamian addressed here: https://github.com/OasisDEX/multiply-proxy-actions/pull/116

**Pull request merged**